TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Library          BuiltIn
+Library          Process
+Library          TAF/testCaseModules/keywords/setup/setup_teardown.py
+Resource         TAF/testCaseModules/keywords/common/commonKeywords.robot
+Suite Setup      Run keywords   Setup Suite
+...                             AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown   Run Teardown Keywords
+Force Tags  skipped
+
+*** Variables ***
+${SUITE}         Export By Kuiper Rules
+${LOG_FILE_PATH}          ${WORK_DIR}/TAF/testArtifacts/logs/export_by_kuiper_rules.log
+
+*** Test Cases ***
+Kuiper001 - Add a new rule and export to MQTT
+    Given Create A Stream
+    And Create A Rule With MQTT Action To Receive Data
+    And Create Device For device-virtual With Name kuiper-mqtt-device
+    When Get device data by device kuiper-mqtt-device and command ${command}
```

Comment:
`When Execute get command ${command} to trigger kuiper-mqtt-device`

Reply:
Updated.